File: apis/backend/src/main/java/com/tms/api/objects/models/organization/OrganizationDetailModel.java
package com.tms.api.objects.models.organization;

import java.util.Date;

import com.tms.api.objects.types.user.UserRoleType;

public record OrganizationDetailModel(
    String organizationId,
    UserRoleType organizationUserRoleType,
    Date createdDate,
    String createdUserId,
    Date modifiedDate,
    String modifiedUserId
) {}

File: ticket-managment-system-web/src/app/core/models/organization/organization-detail.model.ts
import { UserRoleType } from "../../enums";

export class OrganizationDetailModel {
    public organizationId!: string;
    public organizationUserRoleType!: UserRoleType;

    constructor(organizationDetail: Partial<OrganizationDetailModel> = {}) {
        Object.assign(this, organizationDetail);
    }
}